empirical-results.tex

```diff
@@ -1 +1,32 @@
-\section{Empirical Results}
\ No newline at end of file
+\section{Empirical Results}
+
+\newcommand{\rref}[1]{}
+
+In order to evaluate our consumption mechanisms, we implement the federated consumption mechanism in Solidity. We provide a concrete implementation of the \textsf{burn-verifier} contract described in Algorithm~\rref{alg.burn-verifier} as \textsf{BurnedBTC}. We implement the \textsf{Crosschain} parent contract from~\cite{pow-sidechains}. In order to achieve this we implement our own verification methods similar to \rref{alg.verify-event-federation} in our \textsf{Verifier} library. Specifically, we verify transaction data in \textsf{Verifier} similarly to \rref{alg.verify-tx} by making use of the open source bitcoin-spv library~\cite{bitcoin-spv-library}. We verify block connection using a Merkle Mountain Range where each leaf corresponds to a block. The leaf stores the merkle tree root of the merkle tree which collects the transaction id of the transactions included in the block in question.
```

Comment:
Capitalize Merkle Tree

Reply:
Explain what "block connection" means

Reply:
block connection is defined in the previous section